src/tpt_reports/tpt_reports.py

```diff
@@ -67,12 +66,13 @@ def parse_json(data):
     payloads_meta = {}
     try:
         if data:
-            for payload in data["payloads"]:
+            assessment_id = data["id"]
+            for payload in data["phishing_assessment"]["payloads"]:
                 num_payloads += 1
                 payload_data = {}
                 payload_data["Payload"] = payload["payload_description"]
                 # c2_protocol requires additional space to match input file format
-                payload_data["C2 Protocol"] = payload["c2_protocol "]
+                payload_data["C2 Protocol"] = payload["c2_protocol"]
```

Comment:
Does this field name change mean that the comment above this line can now be deleted?

Reply:
Yes. It has been removed in this commit 8d35110a9096931c1fc135340c73351d0755a170.

Reply:
Great, thanks!